citi-onboarding/Delta-odonto
# Feature/our clients react slick

## WHAT I DID
Created the carousel of the clients
## HOW TO TEST
1- On the terminal type:
`cd server`
`node index.js`
2- On another terminal but at the same time type:
`cd client`
`yarn start`
3- Then a page will open on your browser showing the carousel
You will see on the desktop version 2 pictures of two clients and the testimony and name of each client every time you scroll the slide
